dashboard/app/assets/stylesheets/application.scss

```diff
@@ -1730,7 +1730,16 @@ a.download-video {
     margin-bottom: 5px;
     margin-top: 5px;
   }
-  .terms-checkbox, .race-checkbox {
+
+  .terms-container {
+    width: 600px;
+  }
+  .terms-text p {
+    display: inline;
```

Comment:
also here

Reply:
I created a JIRA task and TODO's to resolve this next sprint